smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/XMLHttpResponsePrefixHeaders.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.swift.codegen.integration.httpResponse
+
+import software.amazon.smithy.codegen.core.CodegenException
+import software.amazon.smithy.model.knowledge.HttpBinding
+import software.amazon.smithy.model.shapes.ListShape
+import software.amazon.smithy.model.shapes.MapShape
+import software.amazon.smithy.model.shapes.SetShape
+import software.amazon.smithy.model.shapes.StringShape
+import software.amazon.smithy.swift.codegen.SwiftTypes
+import software.amazon.smithy.swift.codegen.SwiftWriter
+import software.amazon.smithy.swift.codegen.integration.HttpBindingDescriptor
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+
+class XMLHttpResponsePrefixHeaders(
+    val ctx: ProtocolGenerator.GenerationContext,
+    val responseBindings: List<HttpBindingDescriptor>,
+    val writer: SwiftWriter
+) {
+    fun render() {
```

Comment:
Renders statements to set output model values from "prefix style" HTTP headers.
Used to construct a HTTP binding for an output model.